client/src/pages/HomePage/components/HomeGetInvolvedSection/HomeGetInvolvedSection.jsx

```diff
@@ -0,0 +1,30 @@
+import "./HomeGetInvolvedSection.css";
+import { Link } from "react-router-dom";
+import Line from "../../../../components/Line/Line";
+
+import { FaHandHoldingMedical, FaPerson } from "react-icons/fa6";
+
```

Comment:
we can use this for fundraise btn when we click its goes up to the other page
```suggestion
	const handleLinkClick = () => {
		window.scrollTo(0, 0);
	};
```